landing/login/login.controller.js

```diff
@@ -20,5 +20,21 @@
         function activate() {
             $log.debug('Init LoginController ...');
         }
+
+        function login() {
+            let afterSend = function (res) {
+                if (res.token) {
```

Comment:
In response object we always have the next stucture 
`res = { success: [true/false], data: {} }`.
So, in this case you need to check 
`if (res.success) then TokenService.save(res.data.token)`